src/components/update-reminder/update-reminder-content.js

```diff
@@ -9,10 +9,10 @@ export const updateContent = {
+      copy: "You're now able to preview certain datasets in the Metadata panel and expand the preview for greater inspection. If you don't want this enabled, you can turn it off in Settings by selecting the cog icon in the lower left of the screen and toggling the option. In the future, enablement of this feature will be done at a dataset level in Kedro's catalog.yml file.",
```

Comment:
"In the future, enablement of this feature will be done at a dataset level in Kedro's catalog.yml file." isn't very clear what this means. Does it mean, at some point in the future, we will adjust where you set the feature on/off? What is this future point, is it a separate release or ?

I removed the sentence for now because it seems to create more confusion than help (in my mind) but if you want to add it back in, please do so --  just clarify what/when/how